actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -138,13 +139,15 @@ def _filtered_referrer # :nodoc:
       #   link_to "Profiles", controller: "profiles"
       #   # => <a href="/profiles">Profiles</a>
       #
-      # You can use a block as well if your link target is hard to fit into the name parameter. ERB example:
+      # You can use a block as well to wrap your link around a block of HTML. ERB example:
       #
       #   <%= link_to(@profile) do %>
-      #     <strong><%= @profile.name %></strong> -- <span>Check it out!</span>
+      #     <h4><%= @profile.name %></h4>
```

Comment:
I don't see why using a html5 example is a bad think, rails is about current web technologies after all. Also since html5 using the link_to helper this way became the primary use case in all projects that I am involved in. The more important reason for me is: 1. to use an example that uses new multiple lines, which stresses the point of having a block. 2. To get rid of the span-tag which feels a lot like "we just put it there so the example looks more complex" but not for good reasons like semantic or styling.